examples/account_management/link_manager_to_client.py

```diff
@@ -16,13 +16,9 @@
 
 
 import argparse
-import sys
-import uuid
-from datetime import datetime, timedelta
-from google.api_core import protobuf_helpers
 
 from google.ads.google_ads.client import GoogleAdsClient
-from google.ads.google_ads.errors import GoogleAdsException
```

Comment:
Surprised we weren't actually doing any error handling in this example. I filed a bug for us to add it in separately at a later time.